### ISSUE TITLE
Submenu doesn't automatically close when an item is clicked

### ISSUE THREAD

**Issue by cruzjuniel** [AlexTorresSk/custom-electron-titlebar#23]
I noticed that the **.menubar-menu-container** doesn't automatically close when any of its **.action-menu-item** is clicked.

Is there a proper way for this?

I tried:
`$('menubar-menu-button').removeClass('open')`
`$('.menubar-menu-container').remove()`

But since it would still think that the menu is open, I'm getting new issues:
- Hovering above the other menu items will open its submenu
- The submenu will only open after at least 2 clicks.

Any thoughts?

**Comment by ShadyThGod** [comment]
Has there been any progress on this issue?

**Comment by AlexTorresSk** [comment]
> Has there been any progress on this issue?

No I have had time to work on this issue.

If anyone could have time to collaborate on this issue, I would appreciate it.

**Comment by sagargurtu** [comment]
Is there any workaround for this issue?

**Comment by AlexTorresSk** [comment]
I'm currently working on solving this issue.

**Comment by robgonnella** [comment]
Not sure if this is the best approach but I have a working solution here: https://github.com/AlexTorresSk/custom-electron-titlebar/pull/47

**Comment by ahkohd** [comment]
Turns out, you don't need a workaround. This worked but what the heck!


```
let titleBar = new Titlebar({
    backgroundColor: Color.fromHex('#ECECEC')
});

const menu = new Menu();
menu.append(new MenuItem({
    label: 'Item 1',
    submenu: [
        {
            label: 'Subitem 1',
            click: () => {
                console.log('Click on subitem 1');
                 titlebar.updateMenu(menu);
             }
        },
        {
            type: 'separator'
        }
    ]
}));

titlebar.updateMenu(menu);

```

use place``titlebar.updateMenu(menu);` inside your click handler function. wired

**Comment by AlexTorresSk** [comment]
Resolved on #47

**Comment by jonas-johansson** [comment]
I have this issue in `3.2.2-hotfix62`.
